test/tests/test_specs/test_spec_multifile.sh

```diff
@@ -40,6 +40,9 @@ fission fn create --spec --name $fn --env $env --deploy "multifile/*" --entrypoi
 log "Applying specs"
 fission spec apply
```

Comment:
Can we use the --wait flag in this command instead of sleep later?

Reply:
same as above.